projects/schematics/src/add-pwa/index.ts

```diff
@@ -108,9 +90,37 @@ function updateAppModule(options: any): Rule {
 }
 
 export function addPWA(options: SpartacusOptions): Rule {
+  return (tree: Tree, context: SchematicContext) => {
+    const dependencies = [
+      {
+        type: NodeDependencyType.Dev,
+        version: '^0.1001.0',
+        name: '@angular/pwa',
+      },
+    ];
+    return chain([
+      addPackageJsonDependencies(dependencies),
+      invokeAfterAddPWATask(options),
+    ])(tree, context);
+  };
+}
+
+function invokeAfterAddPWATask(options: SpartacusOptions): Rule {
```

Comment:
For the future, there is a similar function in `lib-utils` could we change the shared functions so it can be used for this and future changes of the kind?